src/azure-cli/azure/cli/command_modules/monitor/_params.py

```diff
@@ -357,6 +357,21 @@ def load_arguments(self, _):
     with self.argument_context('monitor log-analytics workspace pack') as c:
         c.argument('intelligence_pack_name', options_list=['--name', '-n'])
         c.argument('workspace_name', options_list='--workspace-name')
+
+    with self.argument_context('monitor log-analytics workspace saved-search') as c:
+        c.argument('saved_search_id', options_list=['--name', '-n'], help='Name of the saved search.')
+        c.argument('workspace_name', options_list='--workspace-name')
+        c.argument('category', help='The category of the saved search. This helps the user to find a saved search faster.')
+        c.argument('display_name', help='Saved search display name.')
+        c.argument('saved_query', options_list=['--saved-query', '-q'], help='The query expression for the saved search.')
+        c.argument('function_alias', options_list=['--func-alias', '--fa'], help='The function alias if query serves as a function.')
```

Comment:
help from Azure portal "Function Aliases are short names given to Saved Searches so they can be easily referenced in query. They are required for Computer Groups."